src/library_wasmfs.js

```diff
@@ -126,6 +126,12 @@ FS.createPreloadedFile = FS_createPreloadedFile;
       })
     },
     // TODO: create
+    create: (path, mode) => {
+      mode = mode !== undefined ? mode : 438 /* 0666 */;
+      mode &= {{{ cDefs.S_IALLUGO }}};
+      mode |= {{{ cDefs.S_IFREG }}};
```

Comment:
Were these settings copied from the JS FS, or do they stem from the docs, or otherwise? A comment would be good here.

Reply:
The settings were copied from the JS FS API and I've added a comment accordingly.